src/api/index.ts

```diff
@@ -99,4 +104,13 @@ export default ({
         },
       },
     },
-  })
+  }
+  // if an agent was provided, bind the agent to the mappersmith configs
+  if (agent) {
+    // gatewayConfigs is not listed as a type on manifest object in mappersmith
+    ;(manifest as any).gatewayConfigs = {
+      configure: () => agent,
```

Comment:
```suggestion
      configure: () => ({ agent }),
```

See here for an example: https://github.com/tulios/mappersmith#http

Reply:
I'm useless without my types (and coffee...). Good catch! Updated.